JEMS-CSE416/JEMS
# implemented delete map api on front-end

**Updates**
- Implemented deleting a map on front-end

**Testing**

**How to review:**
```
git checkout main
git fetch
git checkout delete_map_integration
cd client
npm start
```

1. Go to browser click on a card
2. Click delete map
<img width="1624" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/803e2b5c-b231-413f-b00c-39855ff1ae69">
3. Click confirm
<img width="1625" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/d64b51bf-f0b5-4732-9078-0c920970e0e5">
4. Page should be redirected to homescreen, map should disappear. You can check database as well to be sure
<img width="1621" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/afd40470-ade7-4c84-b6bd-0ec5b7629cec">
